api/preprint_providers/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers as ser
+
+from api.base.utils import absolute_reverse
+from api.base.serializers import JSONAPISerializer, LinksField
+
+
+class PreprintProviderSerializer(JSONAPISerializer):
+
+    filterable_fields = frozenset([
+        'name',
+        'description',
+        'id'
+    ])
+
+    name = ser.CharField(required=True)
+    description = ser.CharField(required=False)
+    id = ser.CharField(max_length=200, source='_id')
+
+    logo_path = ser.CharField(read_only=True)
+    banner_path = ser.CharField(read_only=True)
+
+    links = LinksField({
```

Comment:
This serializer should also give `external_url`, probably as an additional key in `links`